inc/class-sitemaps-provider.php

```diff
@@ -46,24 +46,44 @@ class Core_Sitemaps_Provider {
 	public $slug = '';
 
 	/**
-	 * Get content for a page.
+	 * Get a URL list for a post type sitemap.
 	 *
 	 * @param string $object_type Name of the object_type.
-	 * @param int    $page_num Page of results.
-	 *
-	 * @return int[]|WP_Post[] Query result.
+	 * @param int    $page_num    Page of results.
+	 * @return array $url_list List of URLs for a sitemap.
 	 */
-	public function get_content_per_page( $object_type, $page_num = 1 ) {
-		$query = new WP_Query();
+	public function get_url_list( $object_type, $page_num = 1 ) {
+		$query = new WP_Query( array(
+			'orderby'        => 'ID',
+			'order'          => 'ASC',
+			'post_type'      => $object_type,
+			'posts_per_page' => CORE_SITEMAPS_POSTS_PER_PAGE,
+			'paged'          => $page_num,
+			'no_found_rows'  => true,
+		) );
```

Comment:
It's missing `'post_status' => 'publish'` to prevent exposure of non public content.

Reply:
That's the [default](https://developer.wordpress.org/reference/classes/wp_query/#status-parameters).

We should open an issue to discuss this a bit more in detail:

* What about private posts
* What about attachments (`inherit` post status)

Reply:
Created new issue https://github.com/GoogleChromeLabs/wp-sitemaps/issues/54